File: set_1/easy/python/f7cfcd2c-54dd-4cc8-80cf-6d4b42f2e58d.py
# Solution 1:

def even_odd_rearranger(arr):
    arr_len = len(arr)
    temp_arr = arr_len*[None]
    index = 0
    
    for i in range(arr_len):
        if(arr[i]%2 ==0):
            temp_arr[index] = arr[i]
            index += 1
            
    for i in range(arr_len):
        if(arr[i]%2 != 0):
            temp_arr[index] = arr[i]
            index += 1
    return temp_arr

arr1 = [8, 10, 11, 17, 29, 31, 32, 35, 38, 43]
print(f"The input array --> {arr1}")
res1 = even_odd_rearranger(arr1)
print(f"The array after even-odd re-arrangement --> {res1}")

File: set_1/easy/javascript/f7cfcd2c-54dd-4cc8-80cf-6d4b42f2e58d.js
// Solution 1:
const evenOddRearranger = arr => {
    let tempArr= [];
    let arrLength = arr.length;
    let index = 0;
    for (let i = 0; i < arrLength; i++)
    {
        if (arr[i] % 2 == 0)
        {
            tempArr[index] = arr[i];
            index++;
        }
    }
    for (let i = 0; i < arrLength; i++)
    {
        if (arr[i] % 2 != 0)
        {
            tempArr[index] = arr[i];
            index++;
        }
    }
    return tempArr
}
 

const arr1 = [9, 10, 16, 20, 26, 29, 30, 35, 41, 43];
console.log(`The input array --> [${arr1}]`)

const res1 = evenOddRearranger(arr1);
console.log(res1)